tensorrt_llm/_torch/models/modeling_paligemma.py

```
@@ -0,0 +1,304 @@
+                       ExtraProcessedInputs, MultimodalPlaceholderMetadata,
+                       MultimodalPlaceholderPlacement, TextPrompt,
+                       register_input_processor)
+from ...logger import logger
+from ...sampling_params import SamplingParams
+from ..attention_backend import AttentionMetadata
+from ..model_config import ModelConfig
+from ..modules.linear import Linear
+from .modeling_gemma2 import Gemma2ForCausalLM
+from .modeling_multimodal_utils import fuse_input_embeds
+from .modeling_siglip import SiglipVisionModel
+from .modeling_utils import ModelConfig, filter_weights, register_auto_model
```

Flake8 (7.3.0)

[error] redefinition of unused 'ModelConfig' from line 22

(F811)